MAUI/ImageEditor/keyboard-shortcuts.md

```diff
@@ -0,0 +1,58 @@
+---
+layout: post
+title: Keyboard shortcuts in .NET MAUI Image Editor Control | Syncfusion
+description: Learn here about performing actions using keyboard shortcuts in Syncfusion .NET MAUI Image Editor (SfImageEditor) control.
+platform: maui
+control: SfImageEditor
+documentation: ug
+---
+
+## Keyboard shortcuts in .NET MAUI Image Editor (SfImageEditor)
+
+The [.NET MAUI Image Editor](https://www.syncfusion.com/maui-controls/maui-image-editor) allows the users to navigate through Toolbar items using keyboard shortcuts in Windows and macOS platforms. The below table lists the available shortcuts and the corresponding operations.
```

Comment:
The [Syncfusion .NET MAUI Image Editor](https://www.syncfusion.com/maui-controls/maui-image-editor) supports keyboard shortcuts for user interaction. The table provided outlines the various shortcuts and their associated functions.

Reply:
Updated